keras_nlp/src/tests/test_case.py

```diff
@@ -604,5 +604,27 @@ def compare(actual, expected):
 
             tree.map_structure(compare, output, expected_partial_output)
 
+    def run_pyramid_output_test(
```

Comment:
Can we just add this to the vision backbone test above? `expected_pyramid_output_keys=None` and `if expected_pyramid_output_keys` run everything in this block?

Would make our tests shorter.

Reply:
Done